src/main/kotlin/no/nav/klage/oppgave/domain/klage/Behandling.kt

```diff
@@ -30,17 +27,17 @@ class Behandling(
     val vedtak: Vedtak? = null,
     @OneToOne
     @JoinColumn(name = "eoes_id")
-    val eoes: Eoes,
+    val eoes: Eoes? = null,
     @OneToOne
     @JoinColumn(name = "raadfoert_med_lege_id")
-    val raadfoertMedLege: RaadfoertMedLege,
+    val raadfoertMedLege: RaadfoertMedLege? = null,
     @OneToMany
     @JoinColumn(name = "behandling_id")
     val hjemler: List<Hjemmel>,
     @Column(name = "modified")
-    val modified: LocalDateTime,
+    val modified: LocalDateTime = LocalDateTime.now(),
```

Comment:
Tipper dette vil bli satt av triggere?

Reply:
Ja og nei. Ja den trenger ikke å settes fordi den skal settes av trigger. Og nei, den må settes ellers klager kompilatoren. For å unngå at kompilatoren klager så må den settes til optional, og da reflekterer den ikke datamodellen :)

Reply:
Bruker vi triggere? (Litt skeptisk til det merker jeg..)

Reply:
nei, ikke enda, så vi kan diskutere det :)